test/builder_test.js

```diff
@@ -895,4 +900,113 @@ describe('Builder', function() {
         });
     });
   });
+
+  describe('heimdall stats', function() {
+    it('produces stats', function(done) {
```

Comment:
let use mocha's built in promise support, rather then `done`. This way, error handling is handled for us by default in tests.

This looks like:

```
it('foo', function() {
  return promise.then(() => {});
})
```